Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,20 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/4.9.26.tar.gz"
+  sha256 "28674745db53b6ef1aa4b8466e6e231915dcd596672ec40515d0ab53ee0c33f6"
+
+  depends_on "cmake" => :build
+
+  def install
+    cd "build/cmake"
```

Comment:
Is it necessary to change directory here? What happens if just run in the root of the repo?